src/components/EpisodeCard/EpisodeCard.js

```diff
@@ -0,0 +1,97 @@
+import LazyLoad from 'react-lazyload'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+EpisodeCard.propTypes = {
+  episode: PropTypes.object.isRequired,
+}
+
+export default function EpisodeCard({ episode }) {
+  const { datePublished, duration } = episode
+  const months = [
+    'Jan.',
+    'Feb.',
+    'Mar.',
+    'Apr.',
+    'May.',
+    'Jun.',
+    'Jul.',
+    'Aug.',
+    'Sep.',
+    'Oct.',
+    'Nov.',
+    'Dec.',
+  ]
+  const date = new Date(datePublished * 1000)
+  const time = Math.floor(duration / 60)
+
+  return (
+    <Card>
+      <ImageAndTitleContainer>
+        <LazyLoad height={48} width={48} offset={96} overflow={true} once>
+          <ImageContainer>
+            <Image
+              src={episode.feedImage}
+              alt={'Poster'}
+              width="48"
+              height="48"
+            />
+          </ImageContainer>
+        </LazyLoad>
+
+        <Title>{episode.title}</Title>
+      </ImageAndTitleContainer>
+      <Description>{episode.description}</Description>
+      <Audio controls={true} src={episode.enclosureUrl}></Audio>
+      <Span>{`${date.getDate()} ${
+        months[date.getMonth()]
+      } ${date.getFullYear()} ● ${time === 0 ? 1 : time} min.`}</Span>
```

Comment:
maybe you want to have a look at [toLocaleDateString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString)
using `toLocaleDateString` you also don't need the month array

Reply:
I will look into that again. I thought it was in a different order like month first and so on. But maybe i can alter that.